src/EPR.RegulatorService.Frontend.Web/appsettings.json

```diff
@@ -115,7 +115,8 @@
       "RegistrationSubmissions": "registrations/get-submissions",
       "RegistrationSubmissionDecisionPath": "registration/regulator-decision",
       "FileDownload": "downloads/file-download",
-      "OrganisationRegistrationSubmissionDecisionPath": "organisation-registration-submission-decision"
+      "OrganisationRegistrationSubmissionDecisionPath": "organisation-registration-submission-decision",
+      "GetRegistrationSubmissionDetails": "registrations-submission-details/submissionId/{0}"
```

Comment:
Can this property be renamed to GetOrganisationRegistrationSubmissionDetails

Reply:
it's done